bin/load_upl.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# load the current Uniforme Productnamen Lijst into Open Product
+# Run this script from the root of the repository
+
+set -e
+
+if [[ "${LOAD_UPL,,}" =~ ^(true|1|yes)$ ]]; then
+    # wait for required services
+    /wait_for_db.sh
```

Comment:
Applies to other scripts as well, it might be good to make this relative, that way you can run it locally with `bin/load_upl.sh` as well (see https://github.com/maykinmedia/open-klant/pull/404#discussion_r2079377431)

Reply:
Added, but to use it locally you do need to set LOAD_UPL/RUN_SETUP_CONFIG, DB_HOST & DB_PORT env vars.

Reply:
ah yeah, maybe not that useful then after all, but I guess it doesn't hurt to have it